### PR TITLE
Fix getMode function in AbstractClassContent

### PR DESCRIPTION
**Pull request by fkroockmann**
Currenlty, this function use a old mangement of parameters

### CODE REVIEW

File: ClassContent/AbstractClassContent.php [hunk 1]
```diff
@@ -882,22 +882,8 @@ public function acceptSubcontent($var)
...
-                    case 'select':
-                        $rendermode = $rendermode['selected'];
-                        break;
-                }
-            }
-        }
-
-        return $rendermode;
+        $rendermode = $this->getParamValue('rendermode');
+        return (is_array($rendermode)) ? reset($rendermode) : null;
```
Comment (eric-chau) [inline ClassContent/AbstractClassContent.php]: Follow our coding style:

```php
$rendermode = $this->getParamValue('rendermode');

return is_array($rendermode) ? reset($rendermode) : null;
```

**Comment by eric-chau** [comment]
Nice catch, thank you @fkroockmann :+1:

File: ClassContent/AbstractClassContent.php (hunk 1 above)
Comment (fkroockmann) [inline ClassContent/AbstractClassContent.php]: It's necessary cause the reset function use this param as reference.

Comment (eric-chau) [inline ClassContent/AbstractClassContent.php]: You misunderstood what I said cause I was talking about the coding style but whatever, it's already merged.

**Comment by eric-chau** [comment]
Btw could be really nice to add some tests so we will never get regression about it anymore.

**Pull request merged**